hetmech/tests/test_degree_weight.py

```diff
@@ -636,3 +636,26 @@ def test_dtype(metapath, dtype, dwwc_method):
     metapath = graph.metagraph.metapath_from_abbrev(metapath)
     rows, cols, dwpc_matrix = dwpc(graph, metapath, dtype=dtype, dwwc_method=dwwc_method)
     assert dwpc_matrix.dtype == dtype
+
+
+@pytest.mark.parametrize('metapath,relative', [
+    ('DrDaGiG', 'equal'),
+    ('DaGiGaD', 'equal'),
+    ('DaGaDrDaGaD', 'not_equal'),
+    ('CrCpDrD', 'equal')
+])
+def test_dwpc_approx(metapath, relative):
```

Comment:
Could there be an issue if we change the dense_threshold default of dwpc in the future? Or would these tests work for sparse and dense?

Reply:
These comparisons work either way.